seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/resources/connector-package-service-test-server1-resources/fakesource_to_console.conf

```diff
@@ -38,6 +38,7 @@ source {
 
 sink {
   console {
+    source_table_name = "fake"
```

Comment:
So, without configure `source_table_name`, it will not work fine now?

Reply:
Yes，the current code in `checkComplexGraph ` relies on source_table_name, I changed this pr to draft and find out how to change the logic, but the logic might be same with checkSimpleDAG ~